Automattic/wp-calypso
# e2e: Migrate i18n e2e tests to Playwright

#### Changes proposed in this Pull Request

* Migrate Logged-out Redirect I18n Spec and Gutenboarding I18n Spec to Playwright.
* Remove related components that become unused after migrating to Playwright flows.

#### Testing instructions

* Code review.
* Run tests with `cd test/e2e && LOCALE=fr yarn jest --group=i18n`. Note that `LOCALE` can be any Mag16 locale slug or `en`.

Related to https://github.com/Automattic/wp-calypso/pull/56940
